plugins/examples/tables/src/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { registerResourceTableColumnsProcessor } from '@kinvolk/headlamp-plugin/lib';
+import { ActionButton } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+import Pod from '@kinvolk/headlamp-plugin/lib/K8s/pod';
+import { Menu, MenuItem, Typography } from '@material-ui/core';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function ContextMenu(props: { pod: Pod }) {
+  const { pod } = props;
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const history = useHistory();
+
+  function handleClick(event: any) {
```

Comment:
I don't think it's clear what handleClick is doing. "Our action button" could be verb-ish instead I guess to help understanding?